src/form/fotr.jl

```diff
@@ -48,6 +48,12 @@ function variable_mc_bus_voltage(pm::FOTRUPowerModel; nw=nw_id_default, bounded:
 end
 
 
+"Capacitor switching variables"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature